gossip/agent_test.go

```diff
@@ -69,7 +69,7 @@ func TestLeave(t *testing.T) {
 	conf := DefaultConfig()
 	conf.NodeName = "testNode"
 	conf.Role = member.Auditor
-	conf.BindAddr = "0.0.0.0:12346"
+	conf.BindAddr = "127.0.0.1:12346"
```

Comment:
I would set default bind addr to empty string

Reply:
The tests starts an agent, so it needs a bindAddr. It is in fact changing the DefaultConfig() to have a valid bindaddr. I think we can leave this as it is for now.